Use larger stack-size for the r01109 test. See:

	https://www.varnish-cache.org/trac/ticket/1201

for details.

--- bin/varnishtest/tests/r01109.vtc	2012-08-20 05:20:39.000000000 -0400
+++ bin/varnishtest/tests/r01109.vtc	2012-10-10 09:06:29.000000000 -0400
@@ -22,5 +22,5 @@
 } -start
 
-varnish v1 -arg "-pfetch_chunksize=4k" -arg "-pgzip_level=0" -vcl+backend {
+varnish v1 -arg "-pfetch_chunksize=4k" -arg "-pgzip_level=0" -arg "-pthread_pool_stack=131072" -vcl+backend {
 	sub vcl_fetch {
 		if (req.url ~ "/test") {
